Documentation/Design/Design Doc Template.md

```diff
@@ -0,0 +1,151 @@
+# Feature Name
+
+## Feature Overview
+
+A description of the feature.
+
+
+## Commands
+
+### `command 1`
+
+`command 1 syntax`
+
+A brief description of the command, including any optional parameters or parameters with default values.
+
+### `command 2`
+
+`command 2 syntax`
+
+A brief description of the command, including any optional parameters or parameters with default values.
+
+## Usage
+
+```gls
+command 1 examples
+```
+
+```gls
+command 2 examples
```

Comment:
Do you think it would be good to include the `asdf : asdf asdf` syntax used for these?